### ISSUE TITLE
New Hackage release, and fixing the Nixpkgs package

### ISSUE THREAD

**Issue by evertedsphere** [kmonad/kmonad#537]
The last Hackage release was in 2020, and as a result `haskellPackages.kmonad` has been broken for quite a while due to ghc/dependency version mismatches for `rio` when compiling 0.4.1 that are now fixed in kmonad master.

Would it be possible to cut a new kmonad release?

**Comment by david-janssen** [comment]
Hey @evertedsphere, I'd be happy to help with this. I did the previous hackage release in 2020. Since then @slotThe has really started helping out so much running the show that I am sure he has a better idea of the state of the project than I do. So, I'm going to wait for a few days to see if he has anything to say about this, but when he gives the greenlight I am happy to make the effort.

**Comment by david-janssen** [comment]
Hey @evertedsphere, @slotThe got back to me and we've decided that we're not entirely ready yet for a hackage release. There are some fixes that have been made in `keycode-refactor` that haven't been merged into main yet, so I am going to start working on merging these. Once that is done, however, we will do a release and fix the hackage release. Sadly I can't give you any timeframe on this work since I never know when I'll be healthy enough to work.

**Comment by jokesper** [comment]
There have already been two releases since them. So I will mark this as completed
